sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -32,11 +32,6 @@ if [[ -z ${OUTPUT} ]]; then
     exit 1
 fi
 
-if [[ -z ${TMPDIR} ]]; then
-    echo "TMPDIR is not set"
-    exit 1
-fi
-
```

Comment:
Also, not sure if safe to remove this validation?

Reply:
yup.